### ISSUE TITLE
effect.txt multiplier setting for CreateSparks

### ISSUE THREAD

**Issue by hasteech** [endless-sky/endless-sky#1677]
Too many intensive spark effects strain the CPU and GPU.  Lowering the lifetime reduces GPU strain, but I think there can still be significant CPU load because CreateSparks is still creating and destroying the sparks.  There is already a multiplier at each of the calls to CreateSparks that I saw, so perhaps move this into a setting in effects.txt, so we can mod it.

**Comment by Amazinite** [comment]
https://github.com/Wrzlprnft/endless-sky-no-hit-effects

**Comment by Wrzlprnft** [comment]
The plugin does not change any method calls, just "reduces GPU strain"

**Comment by hasteech** [comment]
Yeah, I wrote something like that plugin, but the CPU load is still there.  It's especially bad for the jump drive, even with lifetime set to 0.
